Automattic/wp-calypso
# BlazePress - Show dates as universal time

Related to #

## Proposed Changes

* Show time in BlazePress without any timezone conversion. The data that displays in campaign information will match server

## Testing Instructions

- Create a campaign
- Check that start date and end date correlates to the campaign you've created
- Set your timezone to US
- check that start date and end date is the same

## Pre-merge Checklist


![image](https://user-images.githubusercontent.com/43957544/221046917-742ef202-581a-4512-808a-9ebc16bb81e5.png)

![image](https://user-images.githubusercontent.com/43957544/221046996-af8384d4-1961-4437-a853-b9b2e15aaf6f.png)
